### PR TITLE
Document agent environment variables

### LINKED ISSUE #1144: SENSU_SUBSCRIPTIONS env var appears to be space delimited, not comma delimited
In testing the sample app and attempting to add a 2nd subscription, the env var `SENSU_SUBSCRIPTIONS` allows for space-delimited strings, not comma-delimited. This should be documented. Example:

<details>
  <summary>Sensu Go K8s Daemonset</summary>

```
apiVersion: extensions/v1beta1
kind: DaemonSet
metadata:
  name: sensu-agent
  labels:
    name: sensu-agent
spec:
  template:
    metadata:
      labels:
        name: sensu-agent
    spec:
      hostPID: true
      hostIPC: true
      hostNetwork: true
      dnsPolicy: ClusterFirstWithHostNet
      containers:
        - name: sensu-agent
          image: sensu/sensu:latest
          command: ["/opt/sensu/bin/sensu-agent", "start", "--log-level", "debug"]
          env:
            - name: HOST_IP
              valueFrom:
                fieldRef:
                  fieldPath: status.hostIP
            - name: SENSU_ID
              value: "kube-node-$(HOST_IP)"
            - name: SENSU_BACKEND_URL
              value: ws://sensu-backend.default.svc.cluster.local:8081
            - name: SENSU_NAMESPACE
              value: demo
            - name: SENSU_SUBSCRIPTIONS
              value: "daemonset proxy"
            - name: SENSU_DEREGISTER
              value: "true"
            - name: SENSU_STATSD_EVENT_HANDLERS
              value: statsd
          ports:
            - containerPort: 3030
              hostPort: 3030
              protocol: TCP
```

</details>

While the above example works, we're using a comma-delimited daemonset in the [classic example here](https://github.com/sensu/sensu-kube-demo/blob/master/classic/deploy/daemonsets/sensu-client-daemonset.yaml#L44-L#45). I'm not sure if this is intended by design. However, this doesn't really seem to jive with the understanding I have of subscriptions being comma-delimited as we see in [the spec attribute examples](https://docs.sensu.io/sensu-go/5.1/reference/entities/#spec-attributes).

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Documents option to use environment variables to configure Sensu agents.

## Motivation and Context

Closes #1144

### CODE REVIEW

File: content/sensu-go/5.2/reference/agent.md [hunk 1]
```diff
@@ -516,6 +516,21 @@ You can specify the agent configuration using a `/etc/sensu/agent.yml` file or u
...
 The agent loads configuration upon startup, so you must restart the agent for any configuration updates to take effect.
 
+Sensu agents can also be configured using environment variables.
+With the exception of `labels`, all agent configuration flags can be applied using environment variables in the format `SENSU_$FLAG_NAME` where `$FLAG_NAME` is the name of the configuration flag.
+Environment variables syntax depends on type; configuration flags of type array must use a space-separated list.
+For example:
+
+{{< highlight shell >}}
+# string | integer | boolean
+export SENSU_NAME=webserver01
```
Comment (apaskulin) [inline content/sensu-go/5.2/reference/agent.md]: remove exports

**Comment by apaskulin** [comment]
This PR requires a follow up to apply changes to version 5.3.

File: content/sensu-go/5.2/reference/agent.md (hunk 1 above)
Comment (asachs01) [reply content/sensu-go/5.2/reference/agent.md]: Maybe add note that this pertains primarily to a container workflow?

File: content/sensu-go/5.2/reference/agent.md [hunk 2]
```diff
@@ -516,6 +516,22 @@ You can specify the agent configuration using a `/etc/sensu/agent.yml` file or u
 See the example config file provided with Sensu at `/usr/share/doc/sensu-go-agent-5.2.1/agent.yml.example`.
 The agent loads configuration upon startup, so you must restart the agent for any configuration updates to take effect.
 
+Sensu agents can also be configured using environment variables.
+Environment variables can be a useful configuration tool for container-based workflows.
```
Comment (apaskulin) [inline content/sensu-go/5.2/reference/agent.md]:
```suggestion
Environment variables can be a useful configuration tool when deploying agents in containers.
```

**Review by rgeniesse — APPROVED** [review]
LGTM.

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**